### DISCUSSION TITLE
Save computed route to KML

### DISCUSSION THREAD

**Discussion by Shohreh** [organicmaps/organicmaps#6382] [Ideas]
Hello,

I suggest making it possible to save the route computed by OM into a KML file, so that it can't be changed by error while holding the phone.

Cheers,

![image](https://github.com/organicmaps/organicmaps/assets/1541861/f4bc2092-741a-438f-9293-34ea3708b7f0)

**Comment by biodranik** [comment]
Please follow https://github.com/organicmaps/organicmaps/issues/1785
